certified-connectors/Talkdesk/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2420 @@
+      "url": "https://support.talkdesk.com/",
+      "email": "support@talkdesk.com"
+    }
+  },
+  "host": "api.talkdeskstg.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/webhooks/schema/{listID}": {
+      "get": {
+        "description": "Schema for selected item in action.",
+        "summary": "Item schema",
+        "operationId": "GetListSchema",
+        "parameters": [
+          {
+            "name": "listID",
+            "type": "string",
+            "in": "path",
+            "description": "Item id.",
+            "required": true,
+            "x-ms-url-encoding": "single",
+            "x-ms-summary": "Item id."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object"
+            }
+          }
+        },
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/contacts/{contact_id}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Data response for this action.",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "contact": {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "description": "Contact ID.",
+                      "title": "Id"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "Contact name.",
+                      "title": "Name"
+                    },
+                    "company": {
+                      "type": "string",
+                      "description": "Contact company.",
+                      "title": "Company"
+                    },
+                    "phones": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "label": {
+                            "type": "string",
+                            "description": "Phone number label.",
+                            "title": "Label"
+                          },
+                          "number": {
+                            "type": "string",
+                            "description": "Phone number in E.164 format.",
+                            "title": "Number"
+                          }
+                        }
+                      },
+                      "description": "The list of the contact's phones.",
+                      "title": "Phones"
+                    }
+                  },
+                  "description": "The contact data.",
+                  "title": "Contact"
+                }
+              }
+            }
+          },
+		  "default": {
```

Comment:
@StefanCRT22 Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
@StefanCRT22 It is not yet resolved, please refer one of the swaggers in our repo and fix the alignment of the properties.

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json